### DISCUSSION TITLE
hot reloading

### DISCUSSION THREAD

**Discussion by poloynikov** [korlibs/korge#925] [Q&A] [answered]
Execution failed for task ':runJvmAutoreload'.
> Could not resolve all files for configuration ':KorgeReloadAgent'.
   > Could not find com.soywiz.korlibs.korge.reloadagent:korge-reload-agent:3.0.0.
     Searched in the following locations:
       - file:/C:/Users/Me/.m2/repository/com/soywiz/korlibs/korge/reloadagent/korge-reload-agent/3.0.0/korge-reload-agent-3.0.0.pom
       - https://repo.maven.apache.org/maven2/com/soywiz/korlibs/korge/reloadagent/korge-reload-agent/3.0.0/korge-reload-agent-3.0.0.pom
       - https://dl.google.com/dl/android/maven2/com/soywiz/korlibs/korge/reloadagent/korge-reload-agent/3.0.0/korge-reload-agent-3.0.0.pom

**Comment by soywiz** [comment, marked as answer]
It seems that the artifact was not published. Now it should be published. https://github.com/korlibs/korge/issues/932
